### PR TITLE
[13.0][MIG] product_variant_configurator

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Adapt Product Variant configurator to v13. Based on #146
CC @ForgeFlow

### CODE REVIEW

File: product_variant_configurator/models/product_configurator.py [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+        product_obj = self.env["product.product"]
+        product = product_obj._product_find(
+            self.product_tmpl_id, self.product_attribute_ids,
+        )
+        if not product:
+            product = product_obj.create(
+                {
+                    "product_tmpl_id": self.product_tmpl_id.id,
+                    "product_template_attribute_value_ids": [
+                        (6, 0, self.product_attribute_ids.mapped("value_id").ids)
```
Comment (ThomasBinsfeld) [inline product_variant_configurator/models/product_configurator.py]: Hello @HviorForgeFlow,

`self.product_attribute_ids.mapped("value_id").ids` will return `product.attribute.value` ids. But `product_template_attribute_value_ids` is a `product.template.attribute.value` field.

We need to build a `product.template.attribute.value` from `self.product_attribute_ids.mapped("value_id")`. I've done a PR with my fix on your fork [here](https://github.com/ForgeFlow/product-variant/pull/1).

WYT?

**Review by ThomasBinsfeld — CHANGES_REQUESTED** [review]

File: product_variant_configurator/models/product_configurator.py (hunk 1 above)
Comment (HviorForgeFlow) [reply product_variant_configurator/models/product_configurator.py]: Looks great!

**Review by ThomasBinsfeld — APPROVED** [review]
Thanks!

**Review by Laurent-Corron — APPROVED** [review]

**Pull request merged**